### ISSUE TITLE
LFS GUI glitches after system upgrade

### ISSUE THREAD

**Issue by Milek7** [iXit/wine-nine-standalone#74]
I don't know what exactly caused this, but after my Arch system upgrade Live for Speed GUI is horribly glitched when running on wine-nine:
![obraz](https://user-images.githubusercontent.com/7935014/80916758-0003a280-8d5b-11ea-9231-268d728e4de9.png)
```
pacman -Q wine-staging wine-nine mesa xf86-video-amdgpu
wine-staging 5.7-1
wine-nine 0.5-1
mesa 20.0.6-2
xf86-video-amdgpu 19.1.0-1
```

**Comment by axeldavy** [comment]
Hum indeed it doesn't look so good.

When was the last time it was working (about which version of mesa) ?

**Comment by Milek7** [comment]
```
[2020-05-03T15:11:36+0200] [ALPM] upgraded wine-staging (5.5-1 -> 5.7-1)
[2020-05-03T15:11:11+0200] [ALPM] upgraded mesa (20.0.4-1 -> 20.0.6-2)
```

https://gist.githubusercontent.com/Milek7/36d2b32ecfe6890bb8557802e24c8c91/raw/3731a80034ce609d81679e8432e3cc15ba60d652/gistfile1.txt

**Comment by axeldavy** [comment]
I checked the mesa commits between 20.0.4 and 20.0.6 and saw nothing suspicious.

I'm not sure what wine-staging could have added that could disturb the game. Btw have you tried disabling csmt (see https://github.com/iXit/wine-nine-standalone/issues/72)

**Comment by axeldavy** [comment]
The problem should be fixed for mesa 20.0.7 which will be released soon
